packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -10540,10 +10540,11 @@ interface ISpinButtonProps {
   max?: number;
   min?: number;
   onBlur?: React.FocusEventHandler<HTMLInputElement>;
+  onChange?: (value: string) => void;
   onDecrement?: (value: string) => string | void;
   onFocus?: React.FocusEventHandler<HTMLInputElement>;
   onIncrement?: (value: string) => string | void;
-  onValidate?: (value: string, event?: React.SyntheticEvent<HTMLElement>) => string | void;
+  onValidate?: (value: string) => string | void;

```

Comment:
Breaking change...

Reply:
There are more big breaking changes on behaviors aspect in this PR. Read this comment: https://github.com/OfficeDev/office-ui-fabric-react/issues/5326#issuecomment-456009631

Reply:
@lijunle is there a way you can leave the event arg there and keep passing it in? What was the motivation to remove

Reply:
The `event` parameter can be added back, but without any sense. The `event` may be *press enter* or *blur*. This `onValidate` is a validator to let the parent pass customized validation logic. Why the parent makes different validation logic depending on which `event` happens? That is completely not making sense.

Reply:
If the changes here significantly change functionality, we could target this fix for the 7.0 branch.

I personally think the control as-is is pretty broken.